### PR TITLE
Final checks

### PR DESCRIPTION
**Pull request by S1hane**

### CODE REVIEW

File: module4/backend/modules/database.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ mongoose.connection.once('open', logConnectionResult);
 
 const schema = {
   name: String,
-  cost: Number,
+  cost: String,
```
Comment (hjlee0704) [inline module4/backend/modules/database.js:29]: any reason why changed from number to string?

File: module4/frontend/dist/bundle.js [hunk 2]
```diff
@@ -130,7 +130,7 @@ var Module1 = /*#__PURE__*/function (_React$Component) {
   _createClass(Module1, [{
     key: "render",
     value: function render() {
-      console.log('rendering Module1');
+      // console.log('Rendering Module1');
```
Comment (hjlee0704) [inline module4/frontend/dist/bundle.js:133]: probably good idea to delete all those commented out console.logs for readability

File: module4/backend/modules/database.js (hunk 1 above)
Comment (S1hane) [reply module4/backend/modules/database.js:29]: As there is no computation taking place that would benefit from a number, this solves a bug that was causing values such as "100.00" to render on the frontend as "100" and numbers like "322.20" to render as "322.2"

File: module4/frontend/src/components/Module1.jsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ class Module1 extends React.Component {
     this.state = {};
   }
   render() {
-    console.log('rendering Module1');
+    // console.log('Rendering Module1');
```
Comment (hjlee0704) [inline module4/frontend/src/components/Module1.jsx:9]: probably better to delete commented out console.logs for better readability

**Review by hjlee0704 — COMMENTED** [review]
i think it is a good idea to brief comments some of functions for better understanding 
nvm

**Pull request merged**